thoth_user_api/utils.py

```diff
@@ -154,6 +156,14 @@ def run_analyzer(image: str, analyzer: str, debug: bool = False,
             f"{registry_user}:{registry_password}"
         )
 
+
```

Comment:
Linter is complaining about the blank line you added - there are 2 blank lines not necessary, delete one.

Reply:
@goern @fridex is there a way to run that locally? 
I have a `pre-commit` [hook](https://github.com/durandom/dotfiles/tree/master/git_template/hooks) that runs rubocop local before committing. Maybe you want to add that to your git hook collection?

Reply:
https://github.com/thoth-station/core/blob/master/doc/development-good-practice.md#install-git-hooks-on-local